plugins/rapid7_insight_agent/icon_rapid7_insight_agent/actions/get_agent_details/action.py

```diff
@@ -16,8 +18,8 @@ def __init__(self):
 
     def run(self, params={}):
         agent_input = params.get(Input.AGENT)
-        agent = self.connection.api.get_agent(agent_input)
+        agent, next_cursor = self.connection.api.get_agent(agent_input)
         # Need to rename agent due to bug in yaml typing
         agent["agent_info"] = agent.pop("agent")
 
-        return {Output.AGENT: insightconnect_plugin_runtime.helper.clean(agent)}
+        return clean({Input.AGENT: agent, Input.NEXT_CURSOR: next_cursor})
```

Comment:
Outputs not inputs 😅 
```suggestion
        return clean({Output.AGENT: agent, Output.NEXT_CURSOR: next_cursor})
```

Reply:
Good catch